Packages/com.unity.inputsystem/Documentation~/PreReleaseNotes.md

```diff
@@ -2,91 +2,33 @@
 
 ## Overview
 
-This pre-release contains updates to the Input System which simplifies and improves some of the main workflows compared with earlier versions of the Input System package.
+This pre-release contains updates to the Input System which simplify and improve some of the main workflows compared with earlier versions of the Input System package. This page describes the main differences introduced, and assumes you are familiar with the workflow in the previous versions of the Input System package.
 
-Because this is a pre-release, the rest of the documentation included with this version of the package has not yet been updated to reflect these changes. The documentation on this page explains the improvements and differences between these new features, and the previous version of the input system package. The improvements are as follows:
+## New project-wide actions
```

Comment:
It seems we do not communicate the min version of Unity for this functionality to be available, maybe state that at least Unity 2022.3 is required?

Reply:
Just the project-wide actions feature (and the new Project Settings UI for it).
Also we don't enable the new UI Toolkit for existing action asset editing and continue using the pre-existing IMGUI based editor.